arches/app/src/components/ControlledListManager/ControlledListSplash.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import { useGettext } from "vue3-gettext";
+
+const props: { description: string } = defineProps(["description"]);
+
+const { $gettext } = useGettext();
+</script>
+
+<template>
+    <div
+        id="rr-splash"
+        class="rr-splash"
```

Comment:
So ``rr-splash`` is just here because the existing core arches css uses it to make the "welcome to ..." splash, and I wasn't planning on reimplementing it.

<img width="520" alt="with-selector" src="https://github.com/archesproject/arches/assets/38668450/909f5404-d851-42a3-8ed5-10590ab09b3e">

I guess OTOH, we could say that now is the time for reimplementing these things.